.vscode/extensions.json

```diff
@@ -1,12 +1,12 @@
 {
-    // See https://go.microsoft.com/fwlink/?LinkId=827846
-    // for the documentation about the extensions.json format
-    "recommendations": [
-        "dzannotti.vscode-babel-coloring",
-        "esbenp.prettier-vscode",
-        "ms-python.python",
-        "dbaeumer.vscode-eslint",
-        "lextudio.restructuredtext",
-        "ziyasal.vscode-open-in-github"
-    ]
+  // See https://go.microsoft.com/fwlink/?LinkId=827846
```

Comment:
We need to figure out how to not get this to happen (probably by placing auto-formatting settings in the repo).